data-raw/mexico_codes.R

```diff
@@ -50,7 +50,7 @@ level_1 <- level_1 %>%
 # Use source data to get level 1 regions INEGI codes
 # - used for matching with level 2
 data_l1 <- check_country_available(country = "Mexico", level = "1")
-data_l1 <- data_l1$download() %>%
+data_l1 <- data_l1$download()[[1]] %>%
```

Comment:
Can we pull this out into its own big fix pr? Hard to keep track of what is happening otherwise!